README.md

```diff
@@ -4,3 +4,18 @@ OnlineSnake.fun is a small fun multiplayer browser game.
 
 ### Background
 This project was started when I was volunteering at the [Junction 2018](https://hackjunction.com/). I wanted to try out real-time web programming with websockets and decided to make a simple game. This is the result.
+
+### Development setup
+#### Docker container
+In project repository, run `docker-compose up`. After the container is ready, navigate to [localhost:8080](http://localhost:8080) using your web browser. If you make changes to web files (client), refresh the web page to see the effect. If you change server files (python), restart the docker container to see changes.
```

Comment:
I think that the Docker container needs to be rebuilt when making changes, since the source files are copied to the container only during the build. Possibly a more detailed description on how to do that could also be added.
```suggestion
In project repository, run `docker-compose up`. After the container is ready, navigate to [localhost:8080](http://localhost:8080) using your web browser. If you make changes to web files (client), refresh the web page to see the effect. If you change server files (python), rebuild and restart the docker container to see changes.
```

Reply:
Thanks for your comments! This is a good point, should definitely commit this change. I would also add the command for rebuild (`docker-compose rebuild`) to the readme.